.changeset/brave-cheetahs-repeat.md

```diff
@@ -0,0 +1,8 @@
+---
+'@aws-amplify/backend': patch
+'@aws-amplify/backend-function': patch
+'@aws-amplify/backend-data': patch
+'@aws-amplify/integration-tests': patch
+---
+
+feat: Add lambda data client
```

Comment:
```suggestion
---
'@aws-amplify/backend': minor
'@aws-amplify/backend-function': minor
'@aws-amplify/backend-data': patch
---

Add lambda data client
```

Explanation:
1. We don't need integration tests here. "Private" packages are excluded from changeset checks.
2. `backend-function` is adding new API. This warrants minor bump.
3. `backend` inherits max of function/data bumps.
4. We're not using conventional commits in our changelogs. See example changelog https://github.com/aws-amplify/amplify-backend/blob/main/packages/backend-function/CHANGELOG.md .